test/SlowTests/Server/Documents/Patching/PatchDebugMode.cs

```diff
@@ -369,7 +369,7 @@ public void DebugResultsWhenDeletingTimeSeries()
                     (deleteTimeSeries[1] as BlittableJsonReaderObject).TryGet("From", out from);
                     Assert.Equal(DateTime.MinValue, from, RavenTestHelper.DateTimeComparer.Instance);
                     (deleteTimeSeries[1] as BlittableJsonReaderObject).TryGet("To", out to);
-                    Assert.Equal(DateTime.MaxValue, to, RavenTestHelper.DateTimeComparer.Instance);
+                    Assert.Equal(DateTime.MaxValue.Millisecond, to.Millisecond);
```

Comment:
it does not make sense? You are comparing only Millisecond component of this 'to' date?

Reply:
done